web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx

```diff
@@ -170,16 +170,20 @@ const FieldCreationModal: React.FC<Props> = ({
         layout="vertical"
         initialValues={initialValues}
         onValuesChange={() => {
-          form
-            .validateFields()
-            .then(() => {
-              setButtonDisabled(false);
-            })
-            .catch(fieldsError => {
-              setButtonDisabled(
-                fieldsError.errorFields.some((item: FieldError) => item.errors.length > 0),
-              );
-            });
+          setTimeout(() => {
+            console.log(form.getFieldsValue());
+
+            form
+              .validateFields()
+              .then(() => {
+                setButtonDisabled(false);
+              })
+              .catch(fieldsError => {
+                setButtonDisabled(
+                  fieldsError.errorFields.some((item: FieldError) => item.errors.length > 0),
+                );
```

Comment:
Do we really need to make this check? If we are in the catch, isn't there a problem?

Reply:
@KaWaite , that's the way validation works in `antd`, it sends the validation errors in the catch statement of `validate` method.

Reply:
So is there a chance we will get to the catch even if there are no errors?

Reply:
@KaWaite , yes anytime any field has a validation missing, it throws an error to the `catch`, so the first block always runs if there are no validation issues, if there is any input with a validation issue it throws it in `catch`.
the checking `fieldsError.errorFields.some` in case any other issue occurs other than the validation issue, as it contains other properties as well, and it may reach the `catch` statement.